src/app/app-routing.module.ts

```diff
@@ -32,7 +35,25 @@ const routes: Routes = [
   },
 
   // Only admin route
-  { path: 'admin', component: AdminPageComponent, canActivate: [adminGuard] },
+  {
+    path: 'admin',
+    component: AdminPageComponent,
+    canActivate: [adminGuard],
+    children: [
+      {
+        path: 'carriages',
+        component: CarriageComponent,
+      },
+      {
+        path: 'routes',
+        component: RoutesComponent,
+      },
+      {
+        path: 'stations',
+        component: CarriageComponent,
```

Comment:
Am I understanding correctly that these are sample components for pages and then we'll post the real ones here?

Reply:
Your component was ready when I started these task. I used it. Routes component will full by me and Artur will add Stations Component